.eslintrc.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
       { terms: ['todo', 'fixme'], location: 'anywhere' },
     ],
     'object-shorthand': 'error',
-    eqeqeq: ['error', 'always'],
+    'eqeqeq': ['error', 'always'],
```

Comment:
Tahle změna a několik vedlejších se nám duplikujou s tím vedlejším PR, že? Nebál bych se to případně vytáhnout bokem do samostatného PR (protože jsou to vesměs drobnosti), ten mergnout přednostně a tyhle PRs nad tím rebasnout, aby obsahovaly jen relevantní změny – ale záleží na tom, jak se ti s tím chce šachovat. Obecně se neboj posílat menší změny jako samostatné PRs, můžem je rychle mergnout.

Reply:
já jsem asi nepustil prettier předtím, to klidně pošlu samostatně